Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
I'm wondering if this might be preferable to using a patch:
```
    mac = <<-EOS.undent
      SOsuf=dylib
      CC = gcc -fpic -pedantic
      LD = gcc
      LDFLAGS = -dynamiclib
      DEBUG=-g -Wno-char-subscripts -O
      SYS_LIBS= -ldl -lm
      PLATFORM_FEATURES= -DUSE_STRLWR=1 -D__APPLE__=1 -DOSX=1
    EOS
    inreplace "makefile",
     "\nFEATURES = $(PLATFORM_FEATURES)",
     "#{mac}\nFEATURES = $(PLATFORM_FEATURES)"
```